app/controllers/users_controller.rb

```diff
@@ -1,4 +1,6 @@
 # :nodoc:
+require 'browser'
```

Comment:
Necessary? Would have expected rails to handle this.
Is there a reason to put this in the users controller?

Reply:
No, I couldn't find it. The community recommends browser gem.

Reply:
I meant the require. If I remove it, I still get log messages like: 
`[3bf5d184-be54-4750-912e-85adfc51c6a2] CSP violation_report | violated_directive=script-src|effective-directive=|disposition=|blocked-uri=inline|document-uri=https://mmt.localtest.earthdata.nasa.gov/collection_drafts/39|source-file=https://mmt.localtest.earthdata.nasa.gov/collection_drafts/39|line-number=864|column-number=1|status-code=|referrer=https://mmt.localtest.earthdata.nasa.gov/collection_drafts/new|browser=Firefox|browser_version=68.0|MMT_version=2a319eae`